clients/nodejs/zpe/Makefile

```diff
@@ -10,16 +10,26 @@ NPM := $(shell command -v npm 2> /dev/null)
 
 ifdef NPM
 
-all:
-	npm install
```

Comment:
For all three "nodejs" projects I did the same change:
1. Replace grunt lint / test calls with regular direct calls from "make" as we did in UI (update package.json + Makefile)
2. Run fix-lint which makes the output a bit different than grunt lint which explains the indentation changes in the project files